src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -254,6 +282,41 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
             return response;
         }
 
+        private async Task<HttpResponseMessage> HandleDeleteHistoryByIdRequestAsync(
+            HttpRequestMessage request,
+            string instanceId)
+        {
+            DurableOrchestrationClientBase client = this.GetClient(request);
+            await client.PurgeInstanceHistoryAsync(instanceId);

```

Comment:
Does `PurgeInstanceHistoryAsync` return a value? I think the DTFx version of the API returns the number of records purged. If so, we should maybe consider including that information in the response. I believe the CLI command Glenna created returns this information.